homeassistant/components/vicare/climate.py

```diff
@@ -162,6 +162,9 @@ def update(self):
                 self._attributes[
                     "boiler_temperature"
                 ] = self._api.getBoilerTemperature()
+                
+                self._attributes["burner_hours"] = self._api.getBurnerHours()
+                self._attributes["burner_starts"] = self._api.getBurnerStarts()
 
             elif self._heating_type == HeatingType.heatpump:
                 self._current_action = self._api.getCompressorActive()
```

Comment:
If you don't mind you could also add
self._attributes["heatpump_supply_temperature"] = self._api.getSupplyTemperaturePrimaryCircuit()
self._attributes["heatpump_return_temperature"] = self._api.getReturnTemperaturePrimaryCircuit()
as requested here: https://community.home-assistant.io/t/viessmann-component/77873/140
Otherwise I'll create a separate PR.

Reply:
Hi @oischinger,

sure, no problem. I've added the lines in the heatpump-else.

Regards,
Stefan

Reply:
@oischinger  @stefanwi87 :  In the current master of PyViCare/Heatpump [Lines 20-30](https://github.com/somm15/PyViCare/blob/master/PyViCare/PyViCareHeatPump.py) we also have two new properties for heatpump statistics: compressor starts and hours, which are similar to burner starts and hours. So, could we add them in this PR as well under the heatpump section? This would give HA users the same functionality/information regardless if they use a burner or heatpump.